content/notebooks/climate_indicators/PAVICStutorial_ClimateDataAnalysis-1DataAccess.ipynb

```diff
@@ -19,9 +19,10 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        return [d for d in files if d.name.split('_')[7] in tcr_likely_models]
</pre><p>Pas sur de mon affaire, mais je trouve ceci un peu plus facile à lire, même si c'est moins "robuste".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-landing/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>